### ISSUE TITLE
Tokenization doesn't work with 0.7.1

### ISSUE THREAD

**Issue by manishrjain** [dgraph-io/dgraph#429]
```
$ curl localhost:8080/query -XPOST -d '{                                                                               ~/benchmarks/data
  me(_xid_: m.06pj8) {
    type.object.name.en
    film.director.film @filter(anyof("type.object.name.en", "war spies"))  {
      _uid_
      type.object.name.en
    }
  }
}' | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   226  100    51  100   175  25487  87456 --:--:-- --:--:-- --:--:--  170k
{
    "me": [
        {
            "type.object.name.en": "Steven Spielberg"
        }
    ]
}
```
No results!

**Comment by jchiu0** [comment]
Went to run the test in tok in embed mode, and found that it wasn't compatible with 58. So I have updated dgraph-io/goicu with version 58 and also pushed the new data file.

The test now passed. Try the following:

```
wget https://github.com/dgraph-io/goicu/raw/master/icudt58l.dat -P /tmp
cd tok
go test . -tags embed -icu /tmp/icudt58l.dat
```

**Comment by pawanrawal** [comment]
@jchiu0 I tried checking filters with the embedded version and I don't think that indexing works still for strings. 

1. I have updated the version of goicu and dgraph. Then I also updated icu to version 58.2 so that the right version is statically linked while creating the embedded binary.
2. After building the embedded Dgraph binary, I started Dgraph like 
`  ./dgraph --schema ~/work/src/github.com/dgraph-io/benchmarks/data/goldendata_schema --icu ~/work/src/github.com/dgraph-io/goicu/icudt58l.dat`
3. Then I loaded up the golden data set.
4. I try to do `anyof` and `allof` queries on `type.object.name.en` listed on our Wiki then and they return empty results for predicates with filters.

Note - `leq`, `geq` queries on `release_date` work
